.github/workflows/ci.yml

```diff
@@ -60,69 +61,22 @@ jobs:
         PYGFX_EXPECT_LAVAPIPE: true
       run: |
         WGPU_FORCE_OFFSCREEN=1 pytest -v tests/
-        pytest -v examples
-        FASTPLOTLIB_NB_TESTS=1 pytest --nbmake examples/notebooks/
-    - uses: actions/upload-artifact@v4
-      if: ${{ failure() }}
-      with:
-        name: screenshot-diffs
-        path: |
-          examples/diffs
-          examples/notebooks/diffs
-
-  test-build-offscreen:
-    name: Test Linux, only offscreen
-    runs-on: ubuntu-latest
-    timeout-minutes: 30
-    if: ${{ !github.event.pull_request.draft }}
-    strategy:
-      fail-fast: false
-      matrix:
-        include:
-          - name: Test py311
-            pyversion: '3.11'
-          - name: Test py312
-            pyversion: '3.12'
-          - name: Test py313
-            pyversion: '3.13'
-    steps:
-    - uses: actions/checkout@v4
-      with:
-        lfs: true
-    - name: Set up Python
-      uses: actions/setup-python@v5
-      with:
-        python-version: ${{ matrix.pyversion }}
-    - name: Install llvmpipe and lavapipe for offscreen canvas
-      run: |
-        sudo apt-get update -y -qq
-        sudo apt-get install --no-install-recommends -y libegl1-mesa-dev libgl1-mesa-dri libxcb-xfixes0-dev mesa-vulkan-drivers xorg-dev
-    - name: Install dev dependencies
-      run: |
-        python -m pip install --upgrade pip setuptools
-        # remove pygfx from install_requires, we install using pygfx@main
-        sed -i "/pygfx/d" ./setup.py
-        pip install git+https://github.com/pygfx/pygfx.git@main
-        pip install -e ".["tests-desktop"]"
-    - name: Show wgpu backend
-      run:
-        python -c "from examples.tests.testutils import wgpu_backend; print(wgpu_backend)"
-    - name: fetch git lfs files
-      run: |
-        git lfs fetch --all
-        git lfs pull
-    - name: Test examples
```

Comment:
git is stupid, this entire block is actually the previous non-notebooks test pipeline that came after afterwards